app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self. hidden = hidden
```

Comment:
There is an extra space after `self.` in `self. hidden`. It should be `self.hidden`.